index.bs

```diff
@@ -4968,6 +4997,7 @@ for their contributions as our W3C Team Contacts.
     "title": "EduPerson Object Class Specification (200604a)",
     "href": "https://www.internet2.edu/media/medialibrary/2013/09/04/internet2-mace-dir-eduperson-200604.html",
     "date": "May 15, 2007"
-  }
+  },
```

Comment:
Delete the common from this line so that it becomes legal JSON.

Reply:
Thanks